clients/ts/signalr/src/Utils.ts

```diff
@@ -1,8 +1,6 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000;
```

Comment:
@BrennanConroy ಠ_ಠ

Reply:
This is product code, not test code :)

Reply:
LOL

Reply:
Should it be moved somewhere else or can it just be removed like this?

Reply:
@BrennanConroy just got overzealous making this change to all the test code and spilled over in to product code.